apps/internal/json/types/time/time.go

```diff
@@ -0,0 +1,32 @@
+package time
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+)
+
+// Unix provides a type that can marshal and unmarshal a string representation
+// of the unix epoch into a time.Time object.
+type Unix struct {
```

Comment:
think this types warrants a line in json/design.md?

Reply:
Nahhh.